pms-migration-utility-default.properties

```diff
@@ -64,7 +68,7 @@ hibernate.generate_statistics=false
 spring.datasource.initialization-mode=never
 
 # websub related properties
-websub.publish.url=mosip.websub.url
+websub.publish.url=${mosip.api.internal.url}/hub/
```

Comment:
why do we use the external URL here why not internal URL ?

Reply:
with that websub url, we are getting an error. Rakshit also reported the same sir. That is why I updated that property

Reply:
we tried with that sir, we used the place holder, still we got the same error.

Reply:
Can you share the exact error here ?

Reply:
Here is the exception that we are getting with that websub url

`io.mosip.kernel.websub.api.exception.WebSubClientException: KER-WSC-103 --> Error occured while publishing topic to hub :- no matching service found for path : /
	at io.mosip.kernel.websub.api.client.PublisherClientImpl.publishUpdate(PublisherClientImpl.java:133)
	at io.mosip.kernel.websub.api.client.PublisherClientImpl.publishUpdate(PublisherClientImpl.java:35)
	at io.mosip.pms.ida.websub.WebSubPublisher.sendEventToIDA(WebSubPublisher.java:44)
	at io.mosip.pms.ida.websub.WebSubPublisher.notify(WebSubPublisher.java:32)
`

Reply:
updated the websub property with the internal URL as suggested sir.